src/internal/m365/collection/teamsChats/handlers.go

```diff
@@ -79,7 +79,7 @@ type includeItemer[I chatsItemer] interface {
 // the given builder.  The tenantID and protectedResourceID are assumed
 // to be stored in the handler already.
 type canonicalPather interface {
-	canonicalPath() (path.Path, error)
+	CanonicalPath() (path.Path, error)
```

Comment:
why is this exported? all references for `canonicalPath` in this PR seems to be in same package.

Reply:
ah, probably a typo on my end.  I'll clean that up.